### DISCUSSION TITLE
1s delay between characters appear since update (iOS)

### DISCUSSION THREAD

**Discussion by roelentless** [standardnotes/forum#3117] [General]
**Describe the bug**
-  Typing has 1 second delay per character, on an iPhone 12 Pro, with a decently sized note. 
-  Font size has changed causing all mobile type notes to wrap across new lines.

**To Reproduce**
1.  Install ios app
2.  Login
3.  Create a decently big note
4.  Type and it takes 1s per character to appear on an iPhone 12

**Expected behavior**
-  Characters appear with less latency.
-  Config to return font size or editor back to old.

**Desktop (please complete the following information):**
 - OS: iOS
 - Version v3.43.5

**Smartphone (please complete the following information):**
 - Device: iPhone12
 - OS: iOS
 - Version: 15.6.1

**Additional context**

The latest app update turned the app from something good into something completely unusable and breaks workflow.
I am an avid user of standard notes, did everything in it, latency before the update was instant.

**Comment by effieeee** [comment]
Hi @ruleb, I'm sorry to hear about that, I noticed you also mentioned how the "font" or the font size was changed. I was wondering if this is happening on **plain text notes** in particular, where a **Monospace font** was made an option to use on the notes?

If so, may I know if you've tried disabling the font from **Preferences > General > Plaintext note defaults**, as shown below?

![image](https://user-images.githubusercontent.com/54989748/196753387-ec55ad32-7818-4148-8dad-f26755ae9148.png)

**Comment by roelentless** [comment]
Hi @effieeee, thanks. This makes the font a bit more usable on mobile (but changes the layout on desktop as well...).

The bigger problem, the high latency between input characters, remains.
